integreat_cms/summ_ai_api/summ_ai_api_client.py

```diff
@@ -80,20 +98,28 @@ async def translate_text_field(self, session, text_field):
                 },
             ) as response:
                 # Wait for the response
-                response_data = await response.json()
-                # Check whether the text was translated successfully
-                if "translated_text" not in response_data:
-                    if "error" in response_data:
-                        raise SummAiException(
-                            f"API error: {response.status} - {response_data['error']}"
-                        )
-                    raise SummAiException(
-                        f"Unexpected API result: {response.status} - {response_data!r}"
+                try:
+                    response_data = await response.json()
+                except (
+                    aiohttp.ContentTypeError,
+                    aiohttp.web_exceptions.HTTPBadRequest,
```

Comment:
It produces AttributeError now at /augsburg/pages/de-si/machine-translate/
`module 'aiohttp' has no attribute 'web_exceptions'`

It requires an import to fix it, but as far as I understand, this exception can't be actually raised here anyway?

Reply:
Aah dang, I forgot the import. I added it because I looked at the implementation, and [it actually seems to be raised if the mime-type doesn't match](https://github.com/aio-libs/aiohttp/blob/54a0d9a3f1fad037c812e75b432054d7c5a9bf45/aiohttp/web_request.py#L674-L681).

Reply:
Your link refers to a different entity.
This is the right one:
https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client_reqrep.py#L1061